test/Sentry.Maui.Tests/MauiEventsBinderTests.cs

```diff
@@ -14,21 +18,29 @@ private class Fixture
 
         public Fixture()
         {
-            var hub = Substitute.For<IHub>();
-            hub.When(h => h.ConfigureScope(Arg.Any<Action<Scope>>()))
+            Hub = Substitute.For<IHub>();
+            Hub.When(h => h.ConfigureScope(Arg.Any<Action<Scope>>()))
                 .Do(c => c.Arg<Action<Scope>>()(Scope));
 
+            Hub.StartSpan(default, default).ReturnsForAnyArgs(args =>
```

Comment:
I don't think you can do this with extension methods. You can mock the stuff they're using in their implementation but extension methods themselves aren't virtual.

Reply:
Fixed